broker/src/main/java/io/zeebe/broker/exporter/metrics/MetricsExporter.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.exporter.metrics;
+
+import io.zeebe.broker.system.configuration.ExporterCfg;
+import io.zeebe.exporter.api.Exporter;
+import io.zeebe.exporter.api.context.Controller;
+import io.zeebe.protocol.record.Record;
+import io.zeebe.protocol.record.RecordType;
+import io.zeebe.protocol.record.ValueType;
+import io.zeebe.protocol.record.intent.JobIntent;
+import io.zeebe.protocol.record.intent.WorkflowInstanceIntent;
+import io.zeebe.protocol.record.value.WorkflowInstanceRecordValue;
+import java.time.Duration;
+import java.util.NavigableMap;
+import java.util.TreeMap;
+import org.agrona.collections.Long2LongHashMap;
+
+public class MetricsExporter implements Exporter {
+
+  public static final Duration TIME_TO_LIVE = Duration.ofSeconds(10);
+  private final ExecutionLatencyMetrics executionLatencyMetrics;
+  private final Long2LongHashMap jobKeyToCreationTimeMap;
+  private final Long2LongHashMap workflowInstanceKeyToCreationTimeMap;
+
+  private final NavigableMap<Long, Long> creationTimeToJobKeyTreeMap;
```

Comment:
I added the TreeMap just to know that we can use these subMap stuff, but yes NavigableMap would also be ok.

> I guess I'm just surprised since we don't really want to care about the concrete time except when creating it man_shrugging

Sorry but I don't get this comment.

Reply:
Sorry, typo, I meant to write `concrete type` not `concrete time` haha, as `treeMap` the concrete type is part of the name :sweat_smile:

Reply:
Haha that makes much more sense😀👍